client/globals.d.ts

```diff
@@ -175,6 +175,11 @@ declare global {
 		timeFormat: string;
 	};
 
+	const woopaymentsPaymentMethodDefinitions: Record<
+		string,
+		PaymentMethodDefinition
```

Comment:
Sorry I didn't notice this in my previous review - my IDE isn't erroring out on it, but where is the `PaymentMethodDefinition` type defined?

Reply:
Ah, good catch. It was resolving to `any` because the `PaymentMethodDefinition` type was in the generated types file in a previous implementation. I've re-added the types to `woopaymentsPaymentMethodDefinitions` in global.d.ts in 6361ad867